api/v1alpha1/loadbalancer_types.go

```diff
@@ -55,6 +55,14 @@ const (
 // load balancer policy
 type ConsistentHash struct {
 	Type ConsistentHashType `json:"type"`
+
+	// The table size for consistent hashing, must be prime number limited to 5000011.
+	//
+	// +kubebuilder:validation:Minimum=1
```

Comment:
should we make it `2`? because `1` is actually not a prime number.

Reply:
Sure, make the minimum to the smallest prime number `2` now.